client/src/components/Stars/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { StarFill, StarHalf, Star } from '@styled-icons/bootstrap';
+
+const StarDiv = styled.div`
+  animation: 800ms ease-in 0s 1 slideInFromTop;
+  margin-top: 5px;
+`;
+
+const Rating = ({ rating, ...props }) => {
+  let starsArr = [];
+  if (rating !== undefined) {
+    while (starsArr.length < 10) {
+      if (rating >= 1) {
+        starsArr.push('StarFill');
+        rating -= 1;
+      } else {
+        if (rating * 10 < 2.5) {
+          starsArr.push('Star');
+        } else if (rating * 10 < 7.5) {
+          starsArr.push('StarHalf');
+        } else {
+          starsArr.push('StarFill');
+        }
+        rating = 0;
+      }
+    }
+  }
+
+  return (
+    <StarDiv>
+      {starsArr.map((star, i) => {
+        if (star === 'StarFill') return <StarFill key={i} />;
+        else if (star === 'Star') return <Star key={i} />;
+        else if (star === 'StarHalf') return <StarHalf key={i} />;
+      })}
+    </StarDiv>
+  );
+};
+
+Rating.prototype = {
```

Comment:
misspelling of propTypes again
```suggestion
Rating.propTypes = {
```